workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt

```diff
@@ -177,10 +179,10 @@ class WorkflowViewStub @JvmOverloads constructor(
     return viewEnvironment[ViewRegistry].buildView(rendering, viewEnvironment, parent)
         .also { newView ->
           if (inflatedId != NO_ID) newView.id = inflatedId
-          newView.visibility = visibility
           background?.let { newView.background = it }
           replaceOldViewInParent(parent, newView)
           actual = newView
+          super.setVisibility(actual.visibility)
```

Comment:
Is this what you meant by 
>what if we always propagate the visibility of actual

I'm guessing the usecase is to allow reads of `actual.visibility` using the view stub as a proxy?

Reply:
No. I meant that when a new view is created, it is assigned the visibility of the previous view — the current `actual`.

Reply:
Basically, change nothing in this class except for line 180, and that becomes:

```
newView.visibility = actual.visibility
```

Reply:
The most minimal change that would fix the issue I was seeing would be to remove the code that sets visibility on actual in `update()`.

The issue occurs when the parent does not want to have any opinion on what the visibility of an inflated workflowViewStub should be.

For example: I have a toast-like view which may or may not be visible after a certain expiration timestamp.

```
data class MapToolTipRendering(
    val text: String,
    val expirationTimeMs: Long?,
)
```

Because the parent always overrides the visibility of the child, it would need to either:
- Know the specifics of how to calculate visibility from a sequence of `MapToolRendering`, which defeats the purpose of `WorkflowViewStub`.
- or, Not override the visibility of `actual`.

```
newView.visibility = actual.visibility
```
The above would make the visibility of `actual` harder to reason about since it would depend on the order of `previous` renderings and their associated runners that are sharing and mutating visibility.